ansible/roles/ocp-workload-3scale-demo/tasks/workload.yml

```
@@ -15,6 +15,10 @@
   set_fact:
     www_project: "www-{{guid}}"
 
+- name: define fuse image stream version
+  set_fact:
+    fuse_version: "application-templates-2.1.fuse-711004-redhat-00003"
```

Can we make this a variable `defaults/main.yml` instead ?

---

updated, please review.
```suggestion
    fuse_version: "application-templates-2.1.fuse-711004-redhat-00003"
```